tests/features/camel-influxdb2/src/main/resources/OSGI-INF/blueprint/route.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<blueprint xmlns="http://www.osgi.org/xmlns/blueprint/v1.0.0"
+           xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+           xmlns:ext="http://aries.apache.org/blueprint/xmlns/blueprint-ext/v1.0.0"
+           xsi:schemaLocation="
+             http://www.osgi.org/xmlns/blueprint/v1.0.0 https://www.osgi.org/xmlns/blueprint/v1.0.0/blueprint.xsd
+             http://camel.apache.org/schema/blueprint http://camel.apache.org/schema/blueprint/camel-blueprint.xsd">
+
+
+    <bean id="myDbClient" class="com.influxdb.client.InfluxDBClientFactory" factory-method="create" >
+        <argument value="http://localhost:$[influxdb2.port]?token=$[admin.token]" />
+    </bean>
+
+
+    <bean id="influxdbconn" class="org.apache.camel.component.influxdb2.InfluxDb2Component">
+        <property name="influxDBClient" ref="myDbClient"/>
+    </bean>
+
+    <ext:property-placeholder placeholder-prefix="$[" placeholder-suffix="]" />
+
+    <camelContext id="myCamelContext" xmlns="http://camel.apache.org/schema/blueprint">
+
+        <route id="pingToInfluxDB2">
+            <from uri="direct:camel-influxdb2-test"/>
+            <to uri="influxdb2:myDbClient?operation=ping&amp;org=test-org&amp;bucket=test-bucket&amp;autoCreateBucket=true" />
+            <setBody>
+                <constant>OK</constant>
+            </setBody>
+            <log message="Ping response: ${body}"/>
+            <to uri="mock:camel-influxdb2-test"/>
+        </route>
+
+        <route id="writeToInfluxDB2">
+            <from uri="direct:camel-influxdb2-write-test"/>
+            <setBody><groovy>def map = ["CamelInfluxDB2MeasurementName" : "cpu", "time": "1307816001290", "idle" : 90L, "user" : 9L]</groovy></setBody>
+            <to uri="influxdb2:myDbClient?operation=insert&amp;org=test-bucket&amp;bucket=test-bucket&amp;autoCreateBucket=true" />
+            <setBody>
+                <constant>OK</constant>
+            </setBody>
```

Comment:
if you want to log the inserted body, this line must be before the setBody part, otherwise, it will just log "OK"